css/index.css

```diff
@@ -143,4 +171,68 @@ footer nav {
 footer nav a {
     color: white;
     text-decoration: none;
+}
+
+
+.jumbo {
+    width: 100%;
+    margin-bottom: 20px;
+}
+
+.middle {
+    width: 800px;
+    margin: auto;
+    display: flex;
+    flex-wrap: wrap;
+}
```

Comment:
1. Perfect use case for `flex-wrap: wrap`, great job!
2. Consider adding `justify-content: space-between` to create separation between your cards.
3. Top and bottom margin can be added to create a space between your content.